hazelcast/src/main/java/com/hazelcast/internal/compatibility/cache/CompatibilityCacheDataSerializerHook.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
```

Comment:
2021 -> 2022? Is it important?

Reply:
Checkstyle will complain for 2022 (`5.0.z` still has the checkstyle rules applicable at time of 5.0 release). I don't think we ever updated checkstyle & copyright dates in any branch other than `master`.